src/CSF.SRDashboard.Client/Services/WorkLoadManagementService.cs

```diff
@@ -347,12 +392,14 @@ private string GetItemDetailFromRequestModel(RequestModel requestModel)
             itemDetail.ProcessingPhase = requestModel.ProcessingPhase;
             itemDetail.HasAttachments = (requestModel.UploadedDocuments != null) ? true : false;
             itemDetail.Comments = requestModel.Comments;
-            itemDetail.Assignment = this.GetAssignmentFromRequestModel(requestModel);
             string itemDetailString = JsonSerializer.Serialize(itemDetail);
 
             return itemDetailString;
         }
 
+        private bool VerifyStatusDuplicate(StatusHistoryItem historyItem, RequestModel requestModel) => string.Equals(requestModel.ProcessingPhase, historyItem.ProcessingPhase) && string.Equals(requestModel.Status, historyItem.StatusText) ? true : false;
```

Comment:
When dealing with booleans you can just send the statement you don't need to use ? true : false
```suggestion
        private bool VerifyStatusDuplicate(StatusHistoryItem historyItem, RequestModel requestModel) => string.Equals(requestModel.ProcessingPhase, historyItem.ProcessingPhase) && string.Equals(requestModel.Status, historyItem.StatusText);
```

Reply:
Fixed